src/lib/middleware/morgan.ts

```diff
@@ -26,12 +26,22 @@ function skipAssets(req: ArtsyRequest, res: ArtsyResponse): boolean {
   )
 }
 
+const BANNED_PARAMS = ["setup_intent_client_secret"]
+export function filterParams(url: string): string {
```

Comment:
Does this need to be exported?

Reply:
Also `filterParams` isn't quite accurate; better something like `maskParams`

Reply:
I was borrowing from [rails' `filter_parameters`](https://api.rubyonrails.org/classes/ActionDispatch/Http/FilterParameters.html) but yeah, i think i agree